From bbfd34724e11bbb1f17c5598fa6202335e619674 Mon Sep 17 00:00:00 2001
From: Michael Hudson-Doyle <michael.hudson@canonical.com>
Date: Wed, 26 Jan 2022 15:50:18 +1300
Subject: [PATCH] clear LC_ALL when setting LC_TIME in rpn2 test

This test was failing when LC_ALL was set to C.UTF-8 and using the C.UTF-8 locale which is part of glibc 2.35, which differs in a very minor way to that that has been used by Debian/Ubuntu for years. The tests sets LC_TIME to C, presumably to try to isolate itself from such things, but that does not have any effect if LC_ALL is set in the build environment.
---
 tests/rpn2 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/rpn2 b/tests/rpn2
index 9f9852f5..b64491d2 100755
--- a/tests/rpn2
+++ b/tests/rpn2
@@ -16,7 +16,7 @@ report "create"
 $RRDTOOL update $RRD 1167487200:0 1167494400:720 1167501600:1440 1167508800:2160 1167516000:2880 1167523200:3600 1167530400:4320 1167537600:5040 1167544800:5760 1167552000:6480 1167559200:7200 1167566400:7920 1167573600:8640 1167580800:9360 1167588000:10080 1167595200:10800 1167602400:11520 1167609600:12240 1167616800:12960
 
 report "update"
-LC_TIME=C TZ=Europe/Zurich $RRDTOOL xport \
+LC_ALL= LC_TIME=C TZ=Europe/Zurich $RRDTOOL xport \
       --json \
       --start  1167487200 --end 1167616800       \
       DEF:myspeed=$RRD:speed:AVERAGE              \
-- 
2.32.0

